src/Microsoft.Data/ColumnArithmeticTemplate.ttinclude

```diff
@@ -114,18 +114,18 @@
         new MethodConfiguration("LeftShift", MethodType.BinaryInt, "<<", isBitwise: true),
         new MethodConfiguration("RightShift", MethodType.BinaryInt, ">>", isBitwise: true),
 
-        new MethodConfiguration("Equals", MethodType.Comparison, "=="),
-        new MethodConfiguration("Equals", MethodType.ComparisonScalar, "=="),
-        new MethodConfiguration("NotEquals", MethodType.Comparison, "!="),
-        new MethodConfiguration("NotEquals", MethodType.ComparisonScalar, "!="),
-        new MethodConfiguration("GreaterThanOrEqual", MethodType.Comparison, ">=", isNumeric:true),
-        new MethodConfiguration("GreaterThanOrEqual", MethodType.ComparisonScalar, ">=", isNumeric:true),
-        new MethodConfiguration("LessThanOrEqual", MethodType.Comparison, "<=", isNumeric:true),
-        new MethodConfiguration("LessThanOrEqual", MethodType.ComparisonScalar, "<=", isNumeric:true),
-        new MethodConfiguration("GreaterThan", MethodType.Comparison, ">", isNumeric:true),
-        new MethodConfiguration("GreaterThan", MethodType.ComparisonScalar, ">", isNumeric:true),
-        new MethodConfiguration("LessThan", MethodType.Comparison, "<", isNumeric:true),
-        new MethodConfiguration("LessThan", MethodType.ComparisonScalar, "<", isNumeric:true),
+        new MethodConfiguration("ElementwiseEquals", MethodType.Comparison, "=="),
```

Comment:
(minor) do we still need the operator here now that we are removing it?

Reply:
Removed